spec/plaindate.html

```diff
@@ -707,18 +707,17 @@ <h1>
           _y2_: an integer,
           _m2_: an integer,
           _d2_: an integer,
-          _largestUnit_: a String,
-        )
+          _largestUnit_: *"year"*, *"month"*, *"week"*, or *"day"*,
```

Comment:
would the conversions be too much if these were spec enums?

Reply:
Both of these are yaks that I'm not ready to shave yet.